app/models.py

```diff
@@ -1467,6 +1467,7 @@ def serialize(self):
         data.update({
             'id': self.id,
             'briefId': self.brief_id,
+            'briefTitle': self.brief.data.get('title', ''),
```

Comment:
It does feel like it would be nicer to serialize the related object into a dictionary, rather than splat its properties all over the parent object?